main.go

```diff
@@ -100,19 +72,37 @@ func main() {
 				cm.Channel.Architecture,
 				cm.Revision,
 				cm.Channel.ReleasedAt.Format(time.Stamp),
-				revisionBuildStatus[cm.Revision],
+				fmt.Sprintf("%d/%d", build.success, build.total),
+				fmt.Sprintf("%d/%d", test.success, test.total),
 			}, table.RowConfig{AutoMerge: true})
 		}
-		t.AppendRow(table.Row{
-			fmt.Sprintf("%s failed %d/%d", testIcon, failedSnapRuns, totalSnapRuns),
-			"", "", "", "", "", "",
-		}, table.RowConfig{AutoMerge: true})
+
 		t.AppendSeparator()
 	}
 
 	t.Render()
 }
 
+type snapStatistcs struct {
+	total, success uint
+	runName        string
+}
```

Comment:
This will not contain snap statistics, but rather Github workflow run stats. `workflowRunStats`?

At the same time, you may want to rename `runs` struct to `workflowRuns` for consistency.

Reply:
Done!